src/themes/material/templates/toolbar-bottom.tpl.html

```diff
@@ -1,3 +1,10 @@
 <ng-template key="bottom-toolbar.tpl.html">
-	<q-grid-pager></q-grid-pager>
+	<div class="q-grid-toolbar">
```

Comment:
please make layout like here: https://github.com/qgrid/ng/blob/master/src/themes/material/templates/toolbar.bottom.html

it will be easier to support